services/movie.py

```diff
@@ -1,11 +1,16 @@
+from typing import Optional
+
+from django.db import transaction
 from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+    genres_ids: Optional[list[int]] = None,
+    actors_ids: Optional[list[int]] = None,
+    title: str = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]

Reply:
Done